docs/hazmat/primitives/asymmetric/dh.rst

```diff
@@ -143,6 +143,9 @@ Key interfaces
 
     .. versionadded:: 0.9
 
+    DH private key objects have methods from this interface as well as
+    :class:`DHPrivateKeyWithSerialization`.
```

Comment:
No, I mean it's very confusing/arguably wrong. Only DH private keys that implement the WithSerialization interface will have those methods; right now that's all the ones we ship, but I don't understand what this is trying to say.